src/general/OptionsMenu.cs

```diff
@@ -99,6 +99,15 @@ public class OptionsMenu : ControlWithInput
     [Export]
     public NodePath GUIMutedPath;
 
+    [Export]
+    public NodePath OutputDeviceSelectionPath;
```

Comment:
I think all the things related to the audio device changing need renaming to be about the *audio* device. "output device" could be something unrelated to audio...